RECOVERY.md

```diff
@@ -119,70 +134,70 @@ bitcoin-cli -rpcwallet=recovery getwalletinfo
 
 You can then find available funds matching the descriptor we imported:
 
-```sh
-bitcoin-cli -rpcwallet=recovery listtransactions
-
+```shell
+ bitcoin-cli -rpcwallet=recovery listtransactions                                                                     
 [
   {
-    "address": "bcrt1qw78cdcsn55vwsvmwe9qgwnx0fwffzqej7keuqfjnwj5xm0f5u6js2hp66f",
+    "address": "bc1p6pxx4mp43xkac222jmfy958gpxqn7duku6cka9ahdfmdp9aak74sza58es",
     "parent_descs": [
-      "wsh(and_v(v:pk(tpubD6NzVbkrYhZ4WnT3E9HUVtswEnituRm6h1m2RQdQH5CWYQGksbns7hx3ediWHpFEkEQC4vPssnQN2gQpzkodRDuMA7nQtWiQ5EDzkGpGVNw/51'/0'/0'),or_d(pk(0256e948180f33f067246710a41656084fc245b97eda081efe1e488b21577d60fd),older(25920))))#m8v4e6vu"
+      "tr(1fc559d9c96c5953895d3150e64ebf3dd696a0b08e758650b48ff6251d7e60d1,and_v(v:pk(xpub6EE2N7jrues5kfjrsyFA5f7hknixqqAEKs8vyMN4QW9vDmYnChzpeBPkBYduBobbe4miQ34xHG4Jpwuq5bHXLZY1xixoGynW31ySUqqVvcU/*),older(2590)))#sv8ug44m"
     ],
     "category": "receive",
-    "amount": 1.50000000,
-    "label": "",
+    "amount": 0.00003000,
     "vout": 1,
-    "confirmations": 5,
-    "blockhash": "6e1048a8d7829d36a766188b499ddcc2e497193427678d115fd341b2b452c0bd",
-    "blockheight": 151,
-    "blockindex": 1,
-    "blocktime": 1687759025,
-    "txid": "d9940b7eb709ff8eaec307bdd6d20633e30a6eb1627d9ef8c8e03dfd28298c75",
-    "wtxid": "261492e5f930b82f65f269bb3006db9c3ef14423e5f52f2a185ace18704bb7b0",
+    "abandoned": false,
+    "confirmations": 0,
+    "trusted": false,
+    "txid": "a9e38fee226e3a598d035afdbecd99c5cb0a6039866cc29fd15d7b27c7d8dcff",
+    "wtxid": "701989d4f18951ae757409ea948e4a9bc3de9bf37dd14a4dcd21ba5355df2401",
     "walletconflicts": [
     ],
-    "time": 1687759025,
-    "timereceived": 1687759181,
-    "bip125-replaceable": "no"
+    "time": 1707745877,
+    "timereceived": 1707745877,
+    "bip125-replaceable": "yes"
   }
 ]
 ```
 
 ### Send funds to a different address
 
 Once those funds have been recovered and the refund delay has expired (the `confirmations` field of the previous command exceeds `25920`), you can send them to your normal on-chain wallet.
-Compute the total amount received (in our example, 1.5 BTC), choose the address to send to (for example, `bcrt1q9ez7rt33wynwpah582lnqlj3u0tpzsrkj2flas`) and create a transaction using all of the received funds:
+For now, this process involves selecting the inputs that you want to spend and creating the spending transaction manually, as documented below, but future versions of Bitcoin Core will probably make this easier.
 
-```sh
-bitcoin-cli -rpcwallet=recovery walletcreatefundedpsbt '[{"txid":"d9940b7eb709ff8eaec307bdd6d20633e30a6eb1627d9ef8c8e03dfd28298c75","vout":1,"sequence":25920}]' '[{"bcrt1q9ez7rt33wynwpah582lnqlj3u0tpzsrkj2flas":1.5}]' 0 '{"subtractFeeFromOutputs":[0]}'
+For example, if `listtransactions` lists a UTXO `5e9d2a387572fe0c8a4996c2f34373b3fbbdb19ff106b84fc91c2450eb27cbe7:0` of `0.002` Bitcoin, this is how you would send it to you another on-chain address.
```

Comment:
```suggestion
For example, if `listtransactions` lists a UTXO `5e9d2a387572fe0c8a4996c2f34373b3fbbdb19ff106b84fc91c2450eb27cbe7:0` of `0.002` Bitcoin, this is how you would send it to another on-chain address.
```

Reply:
fixed in https://github.com/ACINQ/lightning-kmp/pull/563/commits/5e18d0b06f2f6439f8ba2d3c52398f340bcff569